recipes/mcdev/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "mcdev" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/marek-iiasa/mcdev/archive/v{{ version }}.tar.gz
+  sha256: c82d8f2dea0224e8f617384c709871528adbc2e716add511a6c06ad0d4de2bfb
+
+build:
+  entry_points:
+    - mcdev = mcdev:mcma.main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.11,<3.12
+    - setuptools >=61.0
+    - pip
+  run:
+    - python >=3.11,<3.12
+    - pyomo ==6.7.1
+    - pyyaml
+    - seaborn
+    - glpk
+    - dill
+    - numpy ==1.26.4,<2.0.0
```

Do you really want an exact match here? Shouldn't it be the following?
```suggestion
    - numpy >=1.26.4,<2.0.0
```

---

Hi, thank you for the suggestion. I've improved the recipe according to it.